rfc-013-use-docker-to-deploy-and-run-applications-in-containers.md

```diff
@@ -0,0 +1,76 @@
+# Use Docker to deploy and run our applications in containers
+
+## Summary
+
+All of the applications we build are tested and run using Docker containers.
+
+This includes production environments as well as any staging environments and
+continuous integration services.
+
+The decision on whether we MUST use containers in development is not
+included in the scope of this RFC.
+
+## Problem
+
+To date there has been no formal decision to use or not use containers for
+hosting the applications we build. The result of this is a mixed ecosystem that
+is continuously growing.
+
+We currently host our work using numerous services and platforms:
+
+- Govpress
+- AWS
+- Azure
+- Heroku
```

Comment:
Heroku doesn't support Docker containers, does it?

Reply:
I'm not sure when it came out but it does! https://devcenter.heroku.com/categories/deploying-with-docker 

We used it for the first time on Report a Defect.

Reply:
Oh nice!